### PR TITLE
Microsoft.EdgeMarketplace SDK release for GA

### PR DESCRIPTION
**Pull request by aneesh-ponneth**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — DISMISSED** [review]
@aneesh-ponneth:

Please review the Azure SDK onboarding documentation and ensure that your GitHub organization membership and permissions are correct.  

- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You are currently non-compliant because:
  - Your profile does not have a public membership for the `Microsoft` organization
  - You do not have write permissions for the repository

These will need to be corrected before this PR can move forward.

**Comment by aneesh-ponneth** [comment]
> @aneesh-ponneth:
> 
> Please review the Azure SDK onboarding documentation and ensure that your GitHub organization membership and permissions are correct.
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> You are currently non-compliant because:
> 
> * Your profile does not have a public membership for the `Microsoft` organization
> * You do not have write permissions for the repository
> 
> These will need to be corrected before this PR can move forward.

Thanks @jsquire, I have made my Microosft organization public and joined write team. WIll this PR automatically remove the compliance tag?

**Comment by ArthurMa1978** [comment]
Pending on https://github.com/Azure/azure-sdk-pr/issues/1679

**Comment by aneesh-ponneth** [comment]
> > Thanks @jsquire, I have made my Microosft organization public and joined write team.
> 
> Your org memberships look good. You still do not have write permissions in the repository. I'm not sure what group you've joined or whether the membership is still pending, but at this moment, you're still blocked.
> 
> > WIll this PR automatically remove the compliance tag?
> 
> Once I successfully verify the state of your account, I'll remove my blocker and you'll be able to work with Arthur and team on moving this forward.

Hi @jsquire , I have write permission now. Can you please verify and unblock if everything looks good?

**Comment by jsquire** [comment]
> Hi @jsquire , I have write permission now. Can you please verify and unblock if everything looks good?

Looks like everything is good.  Removing my block.   You'll want to work with Arthur and team on the code reivew.

File: sdk/edgemarketplace/Azure.ResourceManager.EdgeMarketplace/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure EdgeMarketplace management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/edgemarketplace/Azure.ResourceManager.EdgeMarketplace/README.md:3]: Update the description of the service by replacing this line.

**Review by ArthurMa1978 — CHANGES_REQUESTED** [review]
Pending on https://github.com/Azure/azure-sdk/issues/6586

**Review by ArthurMa1978 — APPROVED** [review]

**Comment by aneesh-ponneth** [comment]
We want to hold the SDK release as there are some changes in business plan. Will get back when we want to release this

**Pull request closed without merging**